src/commands/schema.ts

```diff
@@ -0,0 +1,60 @@
+import type yargs from "yargs";
+import type { Arguments, CommandBuilder } from "yargs";
+import fetch from "node-fetch";
+import { ChainName } from "@tableland/sdk";
+import getChains from "../chains";
+
+type Options = {
+  // Local
+  name: string;
+
+  chain: ChainName;
+};
+
+export const command = "schema <name>";
+export const desc = "Get info about a given table schema";
+
+export const builder: CommandBuilder = (yargs) =>
+  yargs.positional("name", {
+    type: "string",
+    description: "The target table name",
+  }) as yargs.Argv<Options>;
+
+export const handler = async (argv: Arguments<Options>): Promise<void> => {
+  const { name, chain } = argv;
+
+  // This isn't strictly required, because this REST API is chain agnostic.
+  // But this is how we determine which host string to use.
+  const network = getChains()[chain];
+  if (!network) {
+    console.error("unsupported chain (see `chains` command for details)\n");
+    process.exit(1);
+  }
+
+  const parts = name.split("_");
+  if (parts.length < 3) {
+    console.error(
+      "invalid table name (name format is `{prefix}_{chainId}_{tableId}`)\n"
+    );
+    process.exit(1);
+  }
+
+  // This check isn't strictly required, because this REST API is chain agnostic.
```

Comment:
Same deal here cc @joewagner

Reply:
RE: the comment above, if we chain for host, this check wouldn't be needed right?

Reply:
Correct. We actually _had_ a host optional at some point in the past but dropped it in favor of chain so as to avoid confusion... but maybe its time to reintroduce?

Reply:
What if we reintroduce it, and also keep allowing chain in the methods that are specific to the chain?  Seems like we will want both, but that may depend on the out come of https://github.com/orgs/tablelandnetwork/discussions/240

Reply:
The SDK should support both chain and host connection options right now

Reply:
Agree. I'll leave those design decisions to the outcome of that discussion though. For this PR, I'm going to move along blissfully unaware of that discussion for now. Seem reasonable?

Reply:
For sure! No point in blocking this being used